models/public/forward_tacotron/forward_tacotron_duration_prediction/model.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020 Intel Corporation
```

Comment:
To be consistent with the other composite models, this should be called `forward-tacotron-duration-predictor`.